SilKit/source/services/logging/Logger.cpp

```diff
@@ -22,26 +22,187 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE. */
 #include <chrono>
 #include <iomanip>
 #include <sstream>
+#include <unordered_map>
+#include <string>
 
 #include "Logger.hpp"
 
+#include "StringHelpers.hpp"
+
 #include "fmt/chrono.h"
 #include "fmt/format.h"
 #include "spdlog/spdlog.h"
 #include "spdlog/sinks/null_sink.h"
+
 // NB: we do not use the windows color sink, as that will open "CONOUT$" and
 //     we won't be able to trivially capture its output in SilKitLauncher.
 #include "spdlog/sinks/ansicolor_sink.h"
 #include "spdlog/sinks/stdout_sinks.h"
 #include "spdlog/sinks/basic_file_sink.h"
 
 #include "SpdlogTypeConversion.hpp"
+#include "spdlog/pattern_formatter.h"
+#include "spdlog/fmt/ostr.h"  // support for user defined types
+#include "spdlog/cfg/env.h"   // support for loading levels from the environment variable
```

Comment:
Do we need that header? Is that actually used (loading log-levels from the environment)?